.travis.yml

```diff
@@ -1,18 +1,29 @@
+sudo: false
```

Comment:
Without `sudo: false` line, robot tests fail to complete due to timeouts. Tests still run in new containers.